pkg/proc/target_exec.go

```diff
@@ -89,7 +93,7 @@ func (dbp *Target) Continue() error {
 			}
 			return err
 		}
-		if dbp.StopReason == StopLaunched {
+		if dbp.StopReason == StopLaunched && !dbp.KeepsSteppingBreakpoints {
```

Comment:
The behavior of this is going to be very weird. This code is executed in response to a reverse step request, which results in the recording reaching the beginning of the process execution. 
The reverse step request can never finish if this happens.

Reply:
I think your point about making KeepsSteppingBreakpoints more granular is a good one. I can change this to only apply for halt and tracepoint and take out this check here.